api/hooks/sdtdLogs/index.js

```diff
@@ -62,7 +60,8 @@ module.exports = function sdtdLogs(sails) {
           await sails.hooks.customdiscordnotification.start(serverID);
           return;
         } else {
-          throw new Error(`Tried to start logging for a server that already had it enabled`);
+          const loggingObj = loggingInfoMap.get(serverID);
+          await loggingObj.init();
```

Comment:
there's so much magic going on here it makes me nervious

why is init needed now where error before?

Reply:
Yep true, I also broke a bunch of tests with this so I'll rethink this :smile:

Reply:
Refactored a bit to make it more sane.

This stays though, if something wants to start logging and there is already a loggingObject (that may be stopped0, it shouldnt error but start the existing object